onnxruntime/python/tools/symbolic_shape_infer.py

```diff
@@ -2582,6 +2586,10 @@ def get_prereq(node):
                     get_attribute(node, "then_branch"),
                     get_attribute(node, "else_branch"),
                 ]
+            elif node.op_type == "QuadricCustomOp":
+                # Should have a subgraph, but allow for cases where it's not there
+                subgraph = get_attribute(node, "sub_graph")
+                subgraphs = [subgraph] if subgraph else []
```

Comment:
What happens when there is no subgraph found in the QuadricCustomOp? Do we need to assert or let it be?

Reply:
For now we can let it be. There is some effort required to change over some old ONNXs found in TVM tests, which is why I'm letting it go for now. Since we're not currently using the subgraph for symbolic shape inference, there's no impact here.